src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -405,6 +410,7 @@ QgsPostgresConn::QgsPostgresConn( const QString &conninfo, bool readOnly, bool s
       QgsDebugMsgLevel( "Connecting to " + mUri.connectionInfo( false ), 2 );
       QString connectString = mUri.connectionInfo();
       addDefaultTimeoutAndClientEncoding( connectString );
+      logWrapper = std::make_unique<QgsDatabaseQueryLogWrapper>( QStringLiteral( "PQconnectdb()" ), connectString.toUtf8(), QStringLiteral( "postgres" ), QStringLiteral( "QgsPostgresConn" ), QGS_QUERY_LOG_ORIGIN_PG_CON );
```

Comment:
Maybe useful to add some suffix to "PQconnectdb()"  to differentiate it from the other log entry at the beginning of the function?

Reply:
"libpq::PQconnectdb()" - ?
![0022](https://github.com/qgis/QGIS/assets/31573086/3347d672-e099-4097-943f-55e0dce7adf6)